### PR TITLE
Update to use TPCH scale factor 0.01

### PR DESCRIPTION
**Pull request by ebd2**
Blocked by merge of https://github.com/prestodb/tempto/pull/197 and release of tempto version 1.22

### CODE REVIEW

**Comment by ebd2** [comment]
@electrum 

New expected results files were created by loading TPCH sf 0.01 data into postgresql, running the queries, and fixing the following:
- rounding
- (q17) blank -> null

**Comment by cawallin** [comment]
We think this will fix the intermittent failures on prestodb/master

File: presto-product-tests/src/main/resources/sql-tests/testcases/hive_tpch/q17.result [hunk 1]
```diff
@@ -1,2 +1,2 @@
 -- delimiter: |; ignoreOrder: false; types: DOUBLE 
-348406.05|
+null
```
Comment (maciejgrzybek) [inline presto-product-tests/src/main/resources/sql-tests/testcases/hive_tpch/q17.result]: Why is it `null`?

**Comment by maciejgrzybek** [comment]
Looks good % small comment.

**Comment by maciejgrzybek** [comment]
Though, shouldn't 9333e7d054ceacd92f2f56440f270068d10dde5b state that it's bumping of tempto version?

**Comment by electrum** [comment]
Product tests failed, restarting:
```
error pulling image configuration: Get https://registry-1.docker.io/v2/library/postgres/blobs/sha256:78e3985acac0ede5faff5ffd84584b2278ac47d4c8fdf39f287efd8dd8104ee0: EOF

**Comment by electrum** [comment]
Hive tests failed, too:
```
+docker-compose -f /home/travis/build/prestodb/presto/presto-hive-hadoop2/bin/../conf/docker-compose.yml down
Removing network conf_default
WARNING: Network conf_default not found.
+trap termination_handler INT TERM
+docker-compose -f /home/travis/build/prestodb/presto/presto-hive-hadoop2/bin/../conf/docker-compose.yml pull
Pulling hadoop-master (teradatalabs/cdh5-hive:2.6.0-cdh5.6.0-sshd-4e54a19)...
2.6.0-cdh5.6.0-sshd-4e54a19: Pulling from teradatalabs/cdh5-hive
Digest: sha256:a14e6ff963eeaa3c3bfbf598f78acaa1feae40c1cc385d479aca07cdee5a31e0
Status: Downloaded newer image for teradatalabs/cdh5-hive:2.6.0-cdh5.6.0-sshd-4e54a19
Pulling dnsmasq (teradatalabs/dns:2.75-4e54a19)...
ERROR: received unexpected HTTP status: 503 Service Unavailable

**Comment by electrum** [comment]
The build is failing due to https://github.com/prestodb/presto/issues/6889

**Comment by cawallin** [comment]
@electrum I think those were intermittent dockerhub issues. I rebased and pushed an update which modifies the runHavingWithMultipleAggs query to produce results.

**Comment by ebd2** [comment]
Thanks @cawallin!

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**